client/my-sites/stats/post-performance/index.jsx

```diff
@@ -189,9 +129,12 @@ export default connect( ( state, ownProps ) => {
 	const query = { status: 'published' };
 	const posts = site ? getSitePostsForQuery( state, site.ID, query ) : null;
 	const post = posts && posts.length ? posts[ 0 ] : null;
+	const viewCount = post && site ? getPostStat( state, 'views', site.ID, post.ID ) : null;
+
 	return {
+		viewCount,
 		query,
 		post,
-		loading: isRequestingSitePostsForQuery( state, site.ID, query )
+		loading: ! site || ! posts || viewCount === null
```

Comment:
@gwwar before, we were relying on the query flag (running) to hide or show the stats, With this update I rely on the availability of the data instead, A request may be running in the background to update the displayed data but we apply the principle of "reactivity" and display what we got immediately.